samples/drivers/counter_rtc/src/main.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
```

Comment:
Please move the file under samples/drivers commit 814d929dbc22749a0f19cf98bccdf51c57805296.